Dockerfile.deb-packager

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch-slim
```

Comment:
make the base distro as an ARG (that can default todebian:stretch-slim) so that people have a chance to use an alternative ?

Reply:
Good idea, I'll take a look.